### ISSUE TITLE
Backend default config.yml doesn't work.

### ISSUE THREAD

**Issue by jameskirsop** [untergeek/es_stats_zabbix#9]
I've grabbed the default config file from [here](https://github.com/untergeek/es_stats_zabbix/blob/master/configuration/6.3/config.yml), modified it to suit my environment and tried to run `esz_backend` from the CLI, but I'm getting the following errors:
```
2018-10-23 15:25:43,791 ERROR     Schema error: extra keys not allowed @ data['backend']['apihost']
2018-10-23 15:25:43,792 ERROR     Failed to configure backend: Exception: Configuration: Backend Configuration: Location: backend: Bad Value: "{'cache_timeout': 60, 'apidebug': False, 'apihost': '127.0.0.1', 'apiport': 7600}", extra keys not allowed @ data['backend']['apihost']. Check configuration file.
Failed to configure backend: Exception: Configuration: Backend Configuration: Location: backend: Bad Value: "{'cache_timeout': 60, 'apidebug': False, 'apihost': '127.0.0.1', 'apiport': 7600}", extra keys not allowed @ data['backend']['apihost']. Check configuration file.
```

I've poked around a little in the [schema_check class](https://github.com/untergeek/es_client/blob/master/es_client/helpers/schemacheck.py) and the [config helper](https://github.com/untergeek/es_stats_zabbix/blob/master/es_stats_zabbix/helpers/config.py) but haven't really been able to determine what's going on.

My `backend` section in `config.yml` is unchanged - I've only modified the `elasticsearch` section to provide the IP of the host I want to monitor.

It seems like something is amiss here, but I'm not quite sure what.

**Comment by untergeek** [comment]
Yes, there is an error in the default config.  I haven't fixed it in the code yet.

See [here](https://github.com/untergeek/es_stats_zabbix/blob/master/configuration/6.3/config.yml#L31-L35):
```
backend:
  apihost: 127.0.0.1
  apiport: 7600
  apidebug: false
  cache_timeout: 60
```

The [extra work to strip `api` off the front of those](https://github.com/untergeek/es_stats_zabbix/blob/master/es_stats_zabbix/helpers/config.py#L128-L136) is only for the settings as added from the cli, as illustrated [here](https://github.com/untergeek/es_stats_zabbix/blob/master/es_stats_zabbix/cli/backend.py#L44-L49).

So, I think it should work properly if you change the config file to:
```
backend:
  host: 127.0.0.1
  port: 7600
  debug: false
  cache_timeout: 60
```

**Comment by untergeek** [comment]
I just merged the change.  I'll try to push out an updated release with these changes to PyPI soon (like, this evening).

Fixed by #10
